endpointmanager/pkg/endpointmanager/postgresql/fhirendpointinfostore.go

```diff
@@ -383,7 +385,9 @@ func (s *Store) AddFHIREndpointInfo(ctx context.Context, e *endpointmanager.FHIR
 	}
 
 	var smartResponseJSON []byte
-	if e.SMARTResponse != nil {
+	if e.SMARTResponseBytes != nil {
```

Comment:
I guess you are right that this would never happen, since the object is created from the bytes, I was just making sure I covered all the edge cases but I guess this wouldn't really be one so I will remove it!